### PR TITLE
Fix symlinks in gcc container

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: src/bci_build/package/gcc.py [hunk 1]
```diff
@@ -46,6 +46,7 @@ def _is_main_gcc(os_version: OsVersion, gcc_version: _GCC_VERSIONS) -> bool:
                 (gcc_pkg := f"gcc{gcc_version}"),
                 (gpp := f"{gcc_pkg}-c++"),
                 "make",
+                "gawk",
```
Comment (dcermak) [inline src/bci_build/package/gcc.py:49]: Not objecting, just asking, but this is unrelated to the `-s` flag that you added, right?

**Review by dcermak — APPROVED** [review]

File: src/bci_build/package/gcc.py (hunk 1 above)
Comment (dirkmueller) [reply src/bci_build/package/gcc.py:49]: yes. there was a second commit with the rationale added. "awk" is needed by ./configure.
